### PR TITLE
docs(website): add L3 port setting example for "Run a Taiko node" 

### PR DESCRIPTION
**Pull request by nekokoban-kasoutuu**
add L3 port setting example. When I was setting up, I was very confused.

### CODE REVIEW

File: packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx [hunk 1]
```diff
@@ -145,6 +145,13 @@ For an Eldfell L3 node this would be:
 - `L2_ENDPOINT_HTTP`
 - `L2_ENDPOINT_WS`
 
+If you are running both L2 and L3 node , port will be 8545 for HTTP and 8546 for WSS.
+ex.)
+host.docker.internal:8545
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx:150]: curious: are you using docker for mac/windows or docker for linux? curious if you had to add a:
```
extra_hosts:
    - "host.docker.internal:host-gateway"
```
as some linux users might need to for this to work.

Comment (nekokoban-kasoutuu) [reply packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx:150]: I use linux.
and what you refer to did not need to be set up.

**Comment by nekokoban-kasoutuu** [comment]
What is the progress on this pull request?

File: packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx [hunk 2]
```diff
@@ -145,6 +145,13 @@ For an Eldfell L3 node this would be:
...
+host.docker.internal:8546
+
+If you are accessing the L2 node in different location, the port number will be 8547 and 8548 as in .env file.
+
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node/run-a-taiko-node.mdx:154]:
```suggestion
Take a look at the comments above these values in the `.env.sample` and `.env.sample.l3` to see how to set them up.
```

**Review by dionysuzx — APPROVED** [review]

**Comment by d1onys1us** [comment]
https://github.com/taikoxyz/simple-taiko-node/pull/116 added comments for this step here

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**